dashboard/index.php

```diff
@@ -19,34 +19,46 @@
 
 $Reflector->LoadXML();
 
+function updateHashFile($HashFile, $newLastSync, $newHash) {
+   // Open the file for writing only, If the file does not exist, it is created.
+   // If it exists, it is neither truncated (as opposed to 'w'), nor the call to this function fails
+   // The file pointer is positioned on the beginning of the file. we'll ftruncate() after writing.
+   $Ressource = @fopen($HashFile, "c");

```

Comment:
"c" only works for php 5.2.6 and newer. A lot of people may have still have an older version of php and need a workaround.

Reply:
uhm, thats not good, because in case of disk full we may still end up with empty file :) I'll see what I can do.